llvm/lib/Transforms/Scalar/Reassociate.cpp

```diff
@@ -270,7 +270,7 @@ static BinaryOperator *CreateMul(Value *S1, Value *S2, const Twine &Name,
 }
 
 static Instruction *CreateNeg(Value *S1, const Twine &Name,
-                              Instruction *InsertBefore, Value *FlagsOp) {
+                              BasicBlock::iterator InsertBefore, Value *FlagsOp) {
   if (S1->getType()->isIntOrIntVectorTy())
     return BinaryOperator::CreateNeg(S1, Name, InsertBefore);
```

Comment:
It looks like I missed adding the definition of the iterator-taking CreateNeg in 76dd4bc036, hence this presents in the diff as the argument-type changing instead of the deletion of the `Instruction *` taking function. That's my bad; but shouldn't be the same for the other scenarios where the `Instruction *` taking functions get removed.